api/envoy/admin/v3/config_dump.proto

```diff
@@ -478,3 +478,18 @@ message EndpointsConfigDump {
   // The dynamically loaded endpoint configs.
   repeated DynamicEndpointConfig dynamic_endpoint_configs = 3;
 }
+
+// TODO: Copy this to other versions of the API
+message GenericXdsConfigDump {
+  message XdsResource {
+    string type_url = 1;
```

Comment:
Why do you need a `type_url` here? `Any` already has type_url, and you have type_url in the outer message as well.

Reply:
All these are removed now.